packages/browser/coretest/__tests__/auth/StitchAuthImplUnitTests.ts

```diff
@@ -84,8 +300,10 @@ describe("StitchAuthImpl", () => {
         const [credential] = capture(spiedImpl.loginWithCredentialInternal).last();
 
         expect(credential instanceof StitchAuthCredential).toBeTruthy();
-        expect(credential.providerName).toEqual("test_provider");
-        expect(credential.providerType).toEqual("test_provider");
-        expect((credential as StitchAuthCredential).authInfo).toEqual(redirectResult.ua);        
+        expect(credential.providerName).toEqual("test_provider_name");
+        expect(credential.providerType).toEqual("test_provider_type");
+        expect((credential as StitchAuthCredential).authInfo).toEqual(redirectFragmentResult);
```

Comment:
[request] go a little plus ultra here and mock the profile parts and verify you are logged in with the info above